tools/c7n_mailer/c7n_mailer/slack_delivery.py

```diff
@@ -14,20 +14,18 @@
 import json
 import time
 
-import requests
+from botocore.vendored import requests
```

Comment:
fwiw, the datadog module still depends on requests, so its unclear this really removes a dep, unless we try except our import of it. our lambda packaging subsys is a little baroque for adding package data files, but nutshell certifi pkg's ca pem file needs to be specified by path to make it into the zip.

Reply:
definitely removing slack client removes a few other dependencies (websocket pkg as well), just wanted to note that requests isn't per se one of them.